orchestra/tests/test_dashboard.py

```diff
@@ -304,13 +304,13 @@ def test_submit_entry_level_task_assignment(self):
         # user 0 can't submit a task if its submission prerequisites aren't
         # complete
         step = task.step
-        step.submission_depends_on = [
+        step.submission_depends_on.set([
```

Comment:
good catch---does this happen anywhere else in the codebase?

Reply:
I didn't find any other places where `ManyToManyField` is set using `=` operation. Usually it's done by `_set_step_relations` method